sandbox-tools/merge-repos/src/support/types.ts

```diff
@@ -32,6 +32,12 @@ export declare type CleanupFunc = (git: SimpleGit) => Promise<any>;
      */
     branch: string;
 
+    /**
```

Comment:
Add this optional config (not set for github runs as this defaults to HEAD) to enable more precise local testing